### PR TITLE
MAGECLOUD-2542 Update Cloud welcome

### PR DESCRIPTION
**Pull request by hguthrie**
- Added graphics to Welcome page
- Updated Welcome page text
- Remove subscriptions & Plans, but moved Starter/Pro compare to architecture section
- This nav fix also shows in 2.0, but does not include ece-tools
- I welcome additional suggestions for front page material, provided it is brief
- I did **NOT** remove Cloud from the Setup list on the Main DevDocs page. It would offset the list, so welcome some input on that

Need to complete:
- [x] Run link check-no links associated with these changes
- [x] MAGECLOUD-2543 Move Cloud guide to top navigation bar
- [x] MAGECLOUD-2542 Improve Cloud summary
- [x] Technical review

whatsnew
Update the Welcome page for the [Magento Commerce Cloud guide](https://devdocs.magento.com/guides/v2.2/cloud/bk-cloud.html) and moved the location of the Cloud Guide links to the top navigation bar.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=2872) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=2872) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: hguthrie<br/>:x: Bruce Denham<hr/>**Bruce Denham** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.1/cloud/bk-cloud.md [hunk 1]
```diff
@@ -1,24 +1,29 @@
 ---
 group: cloud
-title: Welcome to Magento Commerce (Cloud)
+title: Magento Commerce Cloud guide
 functional_areas:
   - Cloud
 ---
 
-{{site.data.var.ece}}, formerly Enterprise Cloud Edition, is a managed and automated hosting platform for Magento specifically created for Cloud infrastructures. This version combines {{site.data.var.ee}}, Cloud infrastructure hosting, with a few differences and added features, including Git integration and key environments for development, staging, and live production. You can code, test, and deploy across Integration, Staging, and Production environments for _continuous integration_ of your store.
+{{site.data.var.ece}} is a managed, automated hosting platform for the {{site.data.var.ee}} software. {{site.data.var.ece}} comes with a variety of additional features that sets it apart from the on-prem {{site.data.var.ee}} and {{site.data.var.ce}} platforms:
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/bk-cloud.md]: I wouldn't abbreviate "on-premise".

File: guides/v2.1/cloud/bk-cloud.md [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
 
-Each {{site.data.var.ece}} plan includes an Integration environment for development, testing, and integrating services. Additional active environments are available at additional cost.
+![Cloud Benefits]
 
-Cloud plans include:
+{{site.data.var.ece}} includes features such as git-based **Rapid development** with a pre-provisioned infrastructure that includes PHP, MySQL, Redis, RabbitMQ, and Elasticsearch technologies; **Continuous deployment** every time you push code changes in a PaaS environment; highly **Customizable**  configuration files and tools; and AWS hosting that offers merchants a **Scalable** and **Secure** environment for online sales and retailing.
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/bk-cloud.md]: Should this be a bulleted list?

File: guides/v2.1/cloud/onboarding/onboarding-portal.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ redirect_to:
 The Onboarding Portal gives business and technical users a central location to get started with a {{site.data.var.ece}} project and environments. If you are not sure who the Project Owner is, you can check the Project Owner page in the portal. If you need to change this user, enter a [Support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html).
 
 {:.bs-callout .bs-callout-info}
-The Onboarding Portal is available for new trial accounts and projects for Starter and Pro. Existing projects or non-trial new accounts cannot access the portal at this time. For more information, see [Subscriptions and plans]({{ page.baseurl }}/cloud/basic-information/cloud-plans.html)
+The Onboarding Portal is available for new trial accounts and projects for Starter and Pro. Existing projects or non-trial new accounts cannot access the portal at this time.
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/onboarding/onboarding-portal.md]: I'm pretty sure we discontinued trials and scrubbed references to them in the docs.

Comment (hguthrie) [reply guides/v2.1/cloud/onboarding/onboarding-portal.md]: Yes! Oops. Is it okay to simply remove the word "trial" ? This is a message used in other parts of the docs, so I will update those as well.

File: guides/v2.1/cloud/bk-cloud.md [hunk 4]
```diff
@@ -1,24 +1,29 @@
...
 
--   Active Git branches of code.
--   Unlimited number of *inactive* Git branches.
--   Robust {% glossarytooltip d85e2d0a-221f-4d03-aa43-0cda9f50809e %}Composer{% endglossarytooltip %} build process.
--   And the option to add fully managed services like MySQL, Elasticsearch, Redis, RabbitMQ, and so on, without requiring external add-ons.
+## Technology stack
 
-The {{site.data.var.ece}} architectures use the following technologies:
+Amazon Web Services (AWS) powers the underlying Infrastructure as a Service for {{site.data.var.ece}}. Each {{site.data.var.ece}} plan provides a PaaS Integration environment for developing, testing, and integrating services, see [{{site.data.var.ece}} architecture]({{page.baseurl}}/cloud/architecture/cloud-architecture.html) for a comparison and additional features. Performance tools provide granular visibility into the site performance and PHP profiling. [Fastly]({{page.baseurl}}/cloud/cdn/cloud-fastly.html) caches site assets and, as customers access the site and stores, loads the cached assets fast. Also, the Fastly Web Application Firewall (WAF) prevents malicious traffic and other OWASP Top 10 threats from affecting the site.
```
Comment (bdenham) [inline guides/v2.1/cloud/bk-cloud.md]: Maybe make the parenthetical more explicit using parentheses: "...and integrating services (see Magento Commerce Cloud architecture for...)".

Also, consider moving the details of the tech stack below the graphic. What I've found to work well is to write a brief intro before the graphic--one or two sentences max--to describe the graphic at 1000 ft. Something to prime readers for what they're seeing at avoid putting important details in a place they may skip by jumping right the the graphic. Something like this: "One way to understand the Magento Commerce Cloud is to think of it as five functional layers, as shown here:" Not this: "The following graphic shows an overview of the technology stack", which is already provided by the heading. In general, people typically jump to the graphic and start looking at it, then read further to understand what they're seeing. Highlighting the terms in the paragraph to match the words/phrasing of the graphic is also nice, like you did for the first graphic.

Comment (hguthrie) [reply guides/v2.1/cloud/bk-cloud.md]: Got it. Will move stuff around.

File: guides/v2.1/cloud/live/live-sanity-check.md [hunk 5]
```diff
@@ -36,7 +36,7 @@ Your Git branch must have the following files for building and deploying for you
 
 ## Test build your code locally before pushing {#test-build}
 
-Sometimes you just want to test your build prior to pushing your code to Git. You can use a specific set of commands to build locally. Do **NOT** push the generated build files from this test to your remote Git branch. This is a trial run to ensure no issues occur before pushing to Git. Remember, when you push to the remote Git branch, a full build and deploy process begins automatically.
+Sometimes you just want to test your build prior to pushing your code to Git. You can use a specific set of commands to build locally. Do **NOT** push the generated build files from this test to your remote Git branch. This is a test to ensure no issues occur before pushing to Git. Remember, when you push to the remote Git branch, a full build and deploy process begins automatically.
```
Comment (meker12) [inline guides/v2.1/cloud/live/live-sanity-check.md]: Sometimes you just want to ... is a little fuzzy --   consider making it more directive -- You can use Magento CLI commands to build and test your site before pushing your code to git.

Comment (hguthrie) [reply guides/v2.1/cloud/live/live-sanity-check.md]: Yeah, I didn't write that, for sure. All I did here was remove the word "trial".

File: guides/v2.1/cloud/architecture/cloud-architecture.md [hunk 6]
```diff
@@ -1,12 +1,93 @@
...
+        <ul>
+          <li>All Magento 2 core features</li>
+          <li>PayPal Onboarding Tool</li>
+          <li><a href="https://magento.com/blog/magento-news/introducing-magento-business-intelligence-essentials">Business Intelligence Essentials</a></li>
+          <li><a href="https://magento.com/business-needs/b2b-commerce">B2B module</a></li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>Infrastructure and deployment</td>
```
[the remark this answers is not in the document]
Comment (hguthrie) [reply guides/v2.1/cloud/architecture/cloud-architecture.md]: I didn't touch this table, but I can go back to it and give it a little review and update.

**Comment by hguthrie** [comment]
The recent changes to the DevDocs repository has caused some problems with my navigational changes. This will take a little longer to clean up. Thanks for your patience.

**Comment by hguthrie** [comment]
@bdenham I had to reset to previous commit. Your pushed changes that did not include my nav updates from last night. TSK TSK ! I grabbed the two files before the reset. Thanks for the updates.

**Pull request merged**